aiohttp_cache/backends.py

```diff
@@ -168,41 +142,29 @@ def load_object(self, value: Optional[T] = None) -> Optional[T]:
             return value
 
     async def get(self, key: str) -> Optional[T]:
-        async with self._redis_pool.get() as redis:
-            redis_value = await redis.execute("GET", self.key_prefix + key)
-
-            return self.load_object(redis_value)
+        redis_value = await self._redis.get(self.key_prefix + key)
```

Comment:
why are you not using:
```async with redis.client() as conn:```
like you did in clear method?

Reply:
It's not necessary to acquire connection from the pool for a single command. Clear method use sequence of commands, so that redis.client() is used to acquire connection to make sure all commands are executed on the same connection from the pool.